internal/fs/inode/file.go

```diff
@@ -533,6 +539,12 @@ func (f *FileInode) Write(
 func (f *FileInode) SetMtime(
 	ctx context.Context,
 	mtime time.Time) (err error) {
+	if f.IsLocal() && f.content == nil && f.bwh == nil {
```

Comment:
can we simply check if file is unlinked , dont do anything. 
Gets applied for all files